### ISSUE TITLE
UglifyJS has trouble

### ISSUE THREAD

**Issue by BananaAcid** [valnub/Framework7-Plugin-Welcomescreen#46]
Might be of interest. I tried building the app for production and this showed up:
```
ERROR in app.js from UglifyJs
Unexpected token: punc «(» [./node_modules/f7-welcomescreen/src/f7.welcomescreen.js:233,0][app.js:33458,8]
```

**Comment by valnub** [comment]
Thanks for reporting! Should be fixed in latest [2.1.5 release](https://github.com/valnub/Framework7-Plugin-Welcomescreen/releases/tag/2.1.5). Can you confirm the problem is solved?

**Comment by valnub** [comment]
Closed due to no response
